contracts/levels/Switch.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.24;
+
+contract Switch {
+  bool public switchOn; // switch is off

```

Comment:
It is a detail, but having the variable named 'switchOn followed by 'switch is off' is a bit tangled up. Maybe rename the variable to only 'switch' or 'switchVal' .